llm4pddl/utils.py

```diff
@@ -176,6 +176,33 @@ def get_custom_task(benchmark_name: str, task_num: int) -> Task:
     return Task(domain_file, problem_file)
 
 
+def prep_problem(problem: str) -> str:
+    """Maps a string of a problem file to a new string.
+
+    This new string has had all space removed between grounded atoms.
+    This is for use before querying llm in order to reduce tokens.
+    """
+    # Gets rid of leading and trailing spaces in every line:
+    new = [piece.strip() for piece in problem.split('\n')]
```

Comment:
let's rename this variable something more descriptive, for example, to make clear that it's a list of strings